### PR TITLE
3 RPC Modules Boo.rip, Usercord.org and Nodejs.org

### PR DESCRIPTION
**Pull request by ModBotStudio**
image
image
image
`Boo.rip`: Boo is a elite connection optimizer that aims to improve user gameplay.
`Usercord.org`: Fraud prevention, scam prevention and Discord safety brought to you by UserCord.
`Nodejs.org`: Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine.

### CODE REVIEW

File: Boo.rip/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      "name": "Dominik",
+      "id": "594491170248458250"
+    },
+    "service": "Boo.rip",
+    "description": {
+      "en": "Boo is a elite connection optimizer that aims to improve user gameplay."
+    },
+    "url": "boo.rip",
+    "version": "0.1",
+    "logo": "https://cdn.discordapp.com/attachments/660924113308876820/660924316795404323/boo-official.png",
```
Comment (Bas950) [inline Boo.rip/dist/metadata.json]: Please upload the image somewhere else, dont use discord's cdn

File: Nodejs.org/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      "name": "Dominik",
+      "id": "594491170248458250"
+    },
+    "service": "Nodejs.org",
+    "description": {
+      "en": "Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine."
+    },
+    "url": "nodejs.org",
+    "version": "1.0",
+    "logo": "https://cdn.discordapp.com/attachments/629811375551676429/630434980463640576/usercord_u_logo_tpArtboard_1.png",
```
Comment (Bas950) [inline Nodejs.org/dist/metadata.json]: same here

File: Usercord.org/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      "name": "Dominik",
+      "id": "594491170248458250"
+    },
+    "service": "Usercord.org",
+    "description": {
+      "en": "Fraud prevention, scam prevention and Discord safety brought to you by UserCord."
+    },
+    "url": "usercord.org",
+    "version": "1.0",
+    "logo": "https://cdn.discordapp.com/attachments/629811375551676429/630434980463640576/usercord_u_logo_tpArtboard_1.png",
```
Comment (Bas950) [inline Usercord.org/dist/metadata.json]: and here

File: Boo.rip/dist/metadata.json (hunk 1 above)
Comment (ModBotStudio) [reply Boo.rip/dist/metadata.json]: Done

File: Nodejs.org/dist/metadata.json (hunk 2 above)
Comment (ModBotStudio) [reply Nodejs.org/dist/metadata.json]: Done

File: Usercord.org/dist/metadata.json (hunk 3 above)
Comment (ModBotStudio) [reply Usercord.org/dist/metadata.json]: Done

**Pull request merged**